packages/web-app-admin-settings/src/views/Groups.vue

```diff
@@ -202,7 +202,7 @@ export default defineComponent({
       await loadResourcesTask.perform()
       loadResourcesEventToken = eventBus.subscribe('app.admin-settings.list.load', async () => {
         await loadResourcesTask.perform()
-        selectedGroups.value = []
```

Comment:
setting to new arr broke reactivity in keyboard actions composables

Reply:
sounds fishy 🤔

Reply:
Indeed and I don't get it. As it is a ref, should work out of the box